src/plugins.py

```diff
@@ -35,7 +35,9 @@ def loadPlugin(self, i):
 		else:
 			return False
 		try: disabled = plugin.DISABLED
-		except: plugin.DISABLED = False
```

Comment:
Well, I understand how you feel... On so many levels!  The plugins I first wrote are a __MESS__!  A disgusting revolting vile mess...